client/me/purchases/concierge-banner/index.jsx

```diff
@@ -14,23 +14,78 @@ import { recordTracksEvent } from 'state/analytics/actions';
 import TrackComponentView from 'lib/analytics/track-component-view';
 
 class ConciergeBanner extends Component {
+	getText() {
+		const { bannerType } = this.props;
+
+		let headerText, mainText, buttonText, buttonHref;
+
+		switch ( bannerType ) {
+			case 'upcomingAppointment':
+				headerText = this.props.translate( 'You have an upcoming appointment' );
+				mainText = this.props.translate( 'You have an upcoming Quick Start session appointment.', {
+					comment:
+						"Please extend the translation so that it's clear that these sessions are only available in English.",
+				} );
+				buttonText = this.props.translate( 'View session dashboard' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'View your session dashboard' )` *ES Score: 9*